modules/ROOT/pages/mule-app-dev-helloworld.adoc

```diff
@@ -0,0 +1,193 @@
+and connector operations into a Mule flow. If you do not, review these topics
+in the Studio overview, xref:docs-studio::index.adoc[Anypoint Studio].
+
+== How the Example Works
+
+The Hello Mule example consists of one simple Mule flow. This flow accepts an
+HTTP request, sets a static payload on the message, and then returns a response
+to you.
+
+The request-response inbound HTTP endpoint receives requests the end user
+submits to the Web service. Because the message-exchange pattern is
+request-response, this HTTP endpoint is responsible for both receiving and
+returning messages. The descriptive notes included in the configuration of the
+endpoint can help you understand the actions the endpoint performs.
+
+As its name suggests, the Set Payload component sets a value in the message
+payload. In this example, the value utilizes a Mule expression to set a static
+string on the payload. Like the HTTP endpoint, the configuration contains
+descriptive notes to help you understand what the component does.
+
+
+== Set Up the Example
+
+Follow these steps to create example in your own instance of Studio. Note that
+you can create template applications in Studio and tweak the configurations
+of the use case-based templates to create your own customized applications in
+Mule.
+
+. Create a project in Studio named "HelloMule" by selecting
+*File* > *New* > *Mule Project*, providing the *Project Name*, and clicking
+*Finish*.
++
+Notice that the project name does not contain any blank spaces.
++
+. In the project's *Mule Palette* tab, click *HTTP*, and drag the *Listener*
+operation into the the *Source* section of the *hellomuleFlow*.
+. In the Studio canvas, click the *Listener* to open its configuration UI in
+its *Listener* tab.
++
+. In the listener's *Connector configuration* field, click the *+*, use
+*Test connection* to check that the default configuration works, and then click
+*OK* to add the configuration to the project.
++
+* If the connection is successful, you see the message "Test connection successful!"
+in a *Test connection* dialog that pops up.
+* If the connection fails, try using a port number that is not in use, such as
+`8082` or `8083`, retest the connection, and then add the configuration.
++
+. In the listener's *Path* field, type the path `/hellomule`.
++
+The example uses this path in a web browser to trigger the listener.
++
+. From the *Mule Palette*, click *Core*, and then find and drag a Set Payload
+component to the *Process* area of the Studio canvas.
++
+If you do not find *Set Payload*, try the search field at the top of the
+*Mule Palette*.
++
+. From the Studio canvas, click the Set Payload component to open its
+configuration UI in its *Set Payload* tab.
+. In the component's *Value* field, _deselect_ the *fx* field, and type the
+string `Hello Mule!`, surrounded by quotes.
++
+When you deselect *fx* the value only contains the string `Hello Mule`
+_and does not contain_
+a hash or square brackets.
```

Comment:
done, Just FYI: have to use HTML code &#35; for the hash (in the backticks) due to the way the doc builder interprets it. It's otherwise invisible in the HTML.